File: ios/App/App/KubenavPlugin.m
//
//  KubenavPlugin.m
//  App
//
//  Created by Rico Berger on 07.06.20.
//

#import <Foundation/Foundation.h>
#import <Capacitor/Capacitor.h>

CAP_PLUGIN(KubenavPlugin, "KubenavPlugin",
           CAP_PLUGIN_METHOD(startServer, CAPPluginReturnPromise);
)

File: ios/App/App/KubenavPlugin.swift
//
//  KubenavPlugin.swift
//  App
//
//  Created by Rico Berger on 07.06.20.
//

import Foundation
import Capacitor
import Mobile

@objc(KubenavPlugin)
public class KubenavPlugin: CAPPlugin {
    @objc func startServer(_ call: CAPPluginCall) {
        DispatchQueue.background {
            MobileStartServer()
        }

        call.success()
    }
}
